packages/node_modules/@webex/plugin-rooms/src/rooms.js

```diff
@@ -433,38 +436,50 @@ const Rooms = WebexPlugin.extend({
     try {
       const sdkEvent = cloneDeep(this.eventEnvelope);
       const cluster = getHydraClusterString(webex, activity.object.url);
-      let room;
+      let {tags} = activity.object;
 
       sdkEvent.event = event;
       sdkEvent.data.created = activity.published;
       sdkEvent.actorId = buildHydraPersonId(activity.actor.entryUUID, cluster);
+      if (activity.object.id) {
+        sdkEvent.data.id =
+          buildHydraRoomId(activity.object.id, cluster);
+      }
+      else {
+        sdkEvent.data.id =
+          buildHydraRoomId(activity.target.id, cluster);
+      }
```

Comment:
Any reason this is formatted like? I think it should be fine on the same line.

Reply:
Responding to linter complaints

Reply:
I take it back for these.   These WERE longer when I was using constants and were two lines.  They can be one line now.